src/zjs_event.c

```diff
@@ -443,17 +443,6 @@ void *zjs_event_get_user_handle(jerry_value_t obj)
     return NULL;
 }
 
-static void zjs_event_cleanup(void *native)
-{
-    jerry_release_value(zjs_event_emitter_prototype);
-    zjs_event_initialized = false;
```

Comment:
Instead of this global boolean set the _prototype = 0 initially; when it's set it's its own "existence boolean". And then when it gets freed you'll have to set that global back to 0. (This will mean you'll have to catch the prototype object getting freed, I guess by setting a different native pointer on it.)

Reply:
Fixed.